packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -97,8 +108,10 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 
   public render() {
     let rootClass = 'ms-DatePicker';
-    let { firstDayOfWeek, dateRangeType, strings, isMonthPickerVisible, isDayPickerVisible, autoNavigateOnSelection, showGoToToday, highlightCurrentMonth, navigationIcons } = this.props;
-    let { selectedDate, navigatedDate } = this.state;
+    let { firstDayOfWeek, dateRangeType, strings, showMonthPickerAsOverlay, autoNavigateOnSelection, showGoToToday, highlightCurrentMonth, navigationIcons } = this.props;
+    let { selectedDate, navigatedDate, isMonthPickerVisible, isDayPickerVisible } = this.state;
+    let onHeaderClick = showMonthPickerAsOverlay ? this._onHeaderClick : undefined;
+    let showSmallMonthPicker = isMonthPickerVisible && isDayPickerVisible || showMonthPickerAsOverlay && isMonthPickerVisible ? true : false;
```

Comment:
There were two different types of month pickers in the example.  If a user wants the month picker only (it is the second calendar in the picture below)  it is styled with .pickerOnlyMonthPickerVisible.  It is a large calendar with a grid of 60x60 pixel boxes, and in the css this is the default size.  The other month picker is the smaller one that is beside the daypicker (first calendar in the picture below) it is styled with .pickerIsMonthPickerVisible.  It has a grid of 40x40 pixel boxes that override the default 60 pixel size.  
So, there are two different month picker styles, and I didn't want to adjust the default sizes or anything.

I will use parenthesis and remove the true : false (I added it to be more explicit because all the different month picker styles complicate things)

![cal2](https://user-images.githubusercontent.com/25463697/29326286-c62b56e4-819f-11e7-8ae7-a48651014ec4.jpeg)